src/testsuite/arith128_test_i64.c

```diff
@@ -195,6 +194,66 @@ db_vec_vsrad (vi64_t vra, vui64_t vrb)
 }
 #endif
 
+int
+test_vrld (void)
+{
+  vui64_t i1, i2, e;
+  vui64_t j;
+  int rc = 0;
+
+  printf ("\ntest_vrld Vector Rotate left doubleword\n");
```

Comment:
not sure how pedantic you want to be with capitalization, but this seems inconsistent.
_global_